### PR TITLE
[9.0][MIG] geoengine partner

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: geoengine_partner/README.rst [hunk 1]
```diff
@@ -10,16 +12,29 @@ Geolocalise your partner.
...
 ============
 
-Take a look at the installation section in the description of the module 
+Take a look at the installation section in the description of the module
 'base_geoengine'.
 
+Usage
+=====
+
+Open a partner form and go in `Geoengine Map` tab.
```
Comment (elicoidal) [inline geoengine_partner/README.rst]: Can you use enumerated list? It is easier to follow for SOP

File: geoengine_partner/__openerp__.py [hunk 2]
```diff
@@ -1,25 +1,8 @@
...
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2011-2017 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
 {'name': 'Geospatial support of partners',
- 'version': '8.0.0.1.0',
+ 'version': '9.0.0.1.0',
```
Comment (elicoidal) [inline geoengine_partner/__openerp__.py]: 9.0.1.0.0

File: geoengine_partner/models/partner.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+from openerp.addons.base_geoengine import geo_model
+from openerp.addons.base_geoengine import fields
+
+
+class ResPartner(geo_model.GeoModel):
+    """Add geo_point to partner using a function filed"""
+    _inherit = "res.partner"
+
+    geo_point = fields.GeoPoint('Addresse coordinate')
```
Comment (elicoidal) [inline geoengine_partner/models/partner.py]: Address or Addressee?

**Review by elicoidal — CHANGES_REQUESTED** [review]
some details, no tests

File: geoengine_partner/models/partner.py (hunk 3 above)
Comment (yvaucher) [reply geoengine_partner/models/partner.py]: Oops was Addresses and wanted to remove plural the frenchy way...

**Comment by yvaucher** [comment]
@elicoidal Updated

**Pull request merged**